Update episodes.Rmd (#154)
carpentries/sandpaper-docs: episodes/episodes.Rmd

1. Open the new episode in your editor
2. edit the title
3. add the episode to the `config.yaml`

> 4. preview it with `sandpaper::build_lesson()` or using the <kbd>ctrl + shift + k</kbd> keyboard shortcut.

Reviewer: Oh this is much more clear, thank you!

:::::::::::::::::::::::::
:::::::::::::::::::::::::::::::::::::::::::::::

> To add more challenges, you close the first solution and add more text:

Reviewer: I agree it is a wonky sentence, but it's not that we are adding more challenges, so what about this?

```suggestion
To add more content to the challenge, you close the first solution and add more text:
```

:::::::::::::: callout

### Table alignment best practises

The colons on each side of the `-` in the table dictate how the column is
aligned. By default, columns are aligned left, but if you add colons on either
side, that forces the alignment to that side.

In general, most table contents should be left-aligned, with a couple of 
exceptions:

 - numbers should be right aligned
 - symbols, emojis, and other equal-width items may be center-aligned

These conventions make it easer for folks to scan a table and understand its
contents at a glance. 

::::::::::::::::::::::

> 

Reviewer: this transposition missed the opening callout, so it's swallowed the text and left it with a generic callout title.

Because of the way GitHub reviews works, I cannot suggest edits for that opening fence. Could you bring that fence down to after the table? 

![a large callout block that has "Callout" as its title and contains an example and another section called "Table alignment best practices"](https://github.com/carpentries/sandpaper-docs/assets/3639446/01d7ca5b-0c43-4617-8b1f-cd5cea1b115a)

Author: Whoops-a-doodle! Think I fixed this now.

| orange | :orange_square:  | \$3.09   |
| devil  | :purple_square:  | \$666.00 |

### Table alignment best practises

The colons on each side of the `-` in the table dictate how the column is
aligned. By default, columns are aligned left, but if you add colons on either
side, that forces the alignment to that side.

In general, most table contents should be left-aligned, with a couple of 
exceptions:

 - numbers should be right aligned
 - symbols, emojis, and other equal-width items may be center-aligned

These conventions make it easer for folks to scan a table and understand its
contents at a glance. 

::::::::::::::::::::::

> You can see that we now have a caption associated with the table.

Reviewer: > You can see that we now have a caption associated with the table.

This sentence should still appear above the callout as a weird standalone sentence/paragraph.

:::::::::::::: solution

To get a roughly 1:5:1 ratio, you can use two separators for the short columns
and ten separators for the wide column:

```markdown
Table: example table with overflowing text in three columns

| first | second     | third |

> | --    | :----------: | --    |

Reviewer: The `:` count as characters for length, so the 1:5:1 ratio was correct.

```suggestion
| --    | :--------: | --    |
```

The original produces an HTML table with these widths:

```html
<colgroup>
  <col width="14%">
  <col width="71%">
  <col width="14%">
</colgroup>
```

whereas this modification changes the ratios to be roughly 1:6:1.

```html
<colgroup>
  <col width="12%">
  <col width="75%">
  <col width="12%">
</colgroup>
```

| this should be a small, compact column | this should be a wide column | this column should also be small and compact, much like the first column |
```

Table: example table with overflowing text in three columns

| first | second     | third |

> | --    | :----------: | --    |

Reviewer: ```suggestion
| --    | :--------: | --    |
```

### R Markdown tables

If you are using R Markdown, then you can generate a table from packages like

> {knitr} or {gt}, but make sure to use `results = 'asis'` in your chunk option:

Reviewer: oh good catch! 🙈
